extensions/wikia/CreateNewWiki/tasks/StartPostCreationTasks.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Wikia\CreateNewWiki\Tasks;
+
+class StartPostCreationTasks implements Task {
+
+	use \Wikia\Logger\Loggable;
+
+	/** @var  TaskContext */
+	private $taskContext;
+
+	public function __construct($taskContext) {
+		$this->taskContext = $taskContext;
+	}
+
+	protected function getLoggerContext() {
+		return TaskHelper::getLoggerContext( $this->taskContext );
+	}
+
+	public function prepare() {
+		return TaskResult::createForSuccess();
+	}
+
+	public function check() {
+		return TaskResult::createForSuccess();
+	}
+
+	public function run() {
+		$creationTask = new \Wikia\Tasks\Tasks\CreateNewWikiTask();
+
+		$jobParams = [];
+		$jobParams['url'] = $this->taskContext->getURL();
+		$jobParams['founderId'] = $this->taskContext->getFounder()->getId();
+		$jobParams['founderName'] = $this->taskContext->getFounder()->getName();
+		$jobParams['sitename'] = $this->taskContext->getSiteName();
+		$jobParams['language'] = $this->taskContext->getLanguage();
+		$jobParams['city_id'] = $this->taskContext->getCityId();
+
+		// BugId:15644 - I need to pass this to CreateWikiLocalJob::changeStarterContributions
+		$jobParams->sDbStarter = $this->taskContext->getStarterDb();
```

Comment:
CreateWikiLocalJob has been remove, so either update the comment or move this line